MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -91,7 +92,7 @@
                         not interested in the specific error pass in nil.
  */
 - (id)initWithClientId:(NSString *)clientId
-             authority:(NSString *)authority
+             authority:(MSALAuthority *)authority
```

Comment:
Let's not replace NSString authority with MSALAuthority in public initializers. Or let's at least deprecate the previous initializer and add a new one.